### ISSUE TITLE
help me , thanks!

### ISSUE THREAD

**Issue by LichXichThuy** [fllinares/neural_fingerprints_tf#1]
I ran the run_neural_fingerprints.py file and it crashed like this:
(tensorflow) vu@ubuntu:~/Desktop/neural_fingerprints_tf-master$ python run_neural_fingerprints.py 
usage: run_neural_fingerprints.py [-h] [--val_path VAL_PATH]
                                  [--tst_path TST_PATH]
                                  [--num_epochs NUM_EPOCHS]
                                  [--config_path CONFIG_PATH] [--seed SEED]
                                  train_path out_dir
                                  {hinge,MSE,softmax,sigmoid}
                                  {MSE,RMSE,Corr,0/1,AUC}
                                  [{MSE,RMSE,Corr,0/1,AUC} ...]
run_neural_fingerprints.py: error: too few arguments
